src/store/index.ts

```diff
@@ -76,27 +76,36 @@ const {
     async fetchWorkspaces(context) {
       const { commit } = rootActionContext(context);
       const workspaces = await api.workspaces();
-      commit.setWorkspaces(workspaces);
+      commit.setWorkspaces(workspaces.results.map((h) => h.name));
```

Comment:
I think ideally we'd unwrap the paginated structure of the response, so that `.results` isn't needed, but I'm okay addressing that at a later time.

Reply:
Also, I'm not sure why `h` is used here. While this is very easy to follow, ideally something representative of the data would be used for a variable name (so here, it could be something as simple as `w`, for "workspace").

Reply:
> Also, I'm not sure why `h` is used here. While this is very easy to follow, ideally something representative of the data would be used for a variable name (so here, it could be something as simple as `w`, for "workspace").

Or just `x` so that you don't have to necessarily pick a new letter for every time you use this pattern. But that being said, I think `h` is fine too since the intended semantics is pretty clear.

Reply:
> I think ideally we'd unwrap the paginated structure of the response, so that `.results` isn't needed, but I'm okay addressing that at a later time.

Could you file an issue? This will definitely need to be addressed as soon as we ever have more than one page worth of workspaces.